minecraft/1.7/src/main/java/nova/core/wrapper/mc/forge/v17/wrapper/block/BlockConverter.java

```diff
@@ -136,7 +138,8 @@ private void registerNovaBlock(BlockFactory blockFactory) {
 		FWBlock blockWrapper = new FWBlock(blockFactory);
 		blockFactoryMap.put(blockFactory, blockWrapper);
 		NovaMinecraft.proxy.registerBlock(blockWrapper);
-		GameRegistry.registerBlock(blockWrapper, FWItemBlock.class, blockFactory.getID());
+//		GameRegistry.registerBlock(blockWrapper, FWItemBlock.class, blockFactory.getID());
```

Comment:
It’s impossible with the way NOVA currently handles mod loading.
But once we get rid of Loadable and replace it with something else, it might become possible.